activejob/lib/active_job/serializers.rb

```diff
@@ -4,25 +4,26 @@
 
 module ActiveJob
   # The <tt>ActiveJob::Serializers</tt> module is used to store a list of known serializers
-  # and to add new ones. It also has helpers to serialize/deserialize objects
-  module Serializers
+  # and to add new ones. It also has helpers to serialize/deserialize objects.
+  module Serializers # :nodoc:
```

Comment:
nodoc here will nodoc the ObjectSerializer and this is not what I want.

Reply:
After execution of `bundle exec rake rdoc` i still can see info about `ObjectSerializer`. Shall i remove this `# :nodoc:`?

Reply:
I think it is better. I think it depends on the order the file is parsed.

Reply:
Forget about it. Just saw we don't need to